src/components/Links/Links.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { Icon } from '@edx/paragon';
+
+function Links(props) {
+  const { title, links, className } = props;
+
+  const linkItems = links.map(link => (
+    <li key={link.href}>
+      <Icon className={['fa', 'fa-file', 'mr-2', 'text-primary']} />
+      <a href={link.href} target="_blank" rel="noopener noreferrer">{link.title}</a>
+    </li>
+  ));
+
+  return (
+    <section className={classNames('links', 'mb-3', className)}>
+      <h3 className="mb-3">{title}</h3>
+      <nav>
+        <ul className="list-unstyled">
+          {linkItems}
+        </ul>
+      </nav>
+    </section>
+  );
+}
+
+Links.propTypes = {
+  title: PropTypes.string.isRequired,
+  links: PropTypes.arrayOf(PropTypes.shape({
+    title: PropTypes.string.isRequired,
+    href: PropTypes.string.isRequired,
+  })).isRequired,
+  className: PropTypes.string,
+};
+
+Links.defaultProps = {
+  className: null,
```

Comment:
For an empty default className prop, I typically use `undefined` instead of `null`. When React sees `null` for a className, it still includes an empty `class` attribute on the element in the DOM, whereas `undefined` does not include the `class` attribute at all.